models/gantt.model.project.js

```diff
@@ -1,24 +1,25 @@
-import { Schema, model } from 'mongoose';
- 
+import mongoose, { Schema, Model } from "mongoose";
```

Comment:
There are two types of imports and exports in es6: **default** and **named**. I will suggest you read about them and use only es6 import/export instead of mixing up es5 and es6.

Also I forgot to change var into let and const. If you are using es6, then don't use var. COnvert all of them to let and const.

Reply:
I will read about es6